packages/block-editor/src/components/media-edit/index.native.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import {
+	requestMediaPicker,
+	requestMediaEditor,
+	mediaSources,
+} from 'react-native-gutenberg-bridge';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Picker } from '@wordpress/components';
+
+export const MEDIA_TYPE_IMAGE = 'image';
+
+const editOption = {
+	id: mediaSources.mediaEditor,
```

Comment:
Not sure if we should consider Media Editor as a kind of Image Source. But in the other hand, it will upload a brand new image to the WP Media Library, right? 🤔 

What do you think?

Reply:
I removed it from `mediaSources`.

Yes, you can see it as a media source but in the context of device media library/camera or site library, they're quite not the same thing.

Reply:
>  in the context of device media library/camera or site library, they're quite not the same thing.

Agreed! Thanks for the update